pontoon/base/static/js/translate.js

```diff
@@ -2240,7 +2220,14 @@ var Pontoon = (function (my) {
             success: function(data) {
               if ('count' in data) {
                 var strings = data.count === 1 ? 'string' : 'strings';
-                message = data.count + ' ' + strings + ' ' + action + 'd';
+                var actionText = action + 'd';
+
+                if (action === 'reject') {
+                  itemsText = 'suggestion' + (data.count === 1 ? '' : 's');
```

Comment:
You should initialize `itemsText`, otherwise it will raise a `ReferenceError` on any action that is not *reject*.